### DISCUSSION TITLE
Offline Access without using Smart Sync on GoogleDrive MacOS

### DISCUSSION THREAD

**Discussion by makeitrad** [iterate-ch/cyberduck#14747] [Q&A]
Is it possible to use Offline access without using Smart Sync? Our google drive has 100's of GBs worth of data and Smart Sync goes into this huge download sequence that will last for hours and hours, downloading un-needed files that I havn't set for offline access. It also ties up mass amounts of space on my boot drive. Upwards of 50GB that isn't located where my Cache is set in preferences. Any help appreciated!!! Thank you!

**Comment by tatze96** [comment]
The offline feature is only available within the *Smart Synchronization* mode.

Mountain Duck generally won't download files by itself unless they are opened by the user, marked to keep offline on local disk, or got opened or, respectively, read through by an external third-party application like an antivirus application.

Regarding the cache location:
The new cache location doesn't apply right away for presently connected bookmarks. Bookmarks that were connected while changing the cache location need to be reconnected to use the new cache location.

The previous cache location won't get cleared out automatically. Deleting the cache folder can lead to file loss if there are synchronization errors, conflicts, or files that aren't fully synchronized yet. You can delete the previous _Cache_ folder if you are sure that all files are present.
